tests/integration_tests/functional/test_cmd_line_start.py

```diff
@@ -104,3 +104,41 @@ def test_config_start_no_api(test_microvm_with_api, vm_config_file):
         exceptions=RuntimeError,
         tries=10,
         delay=1)
+
+
+@pytest.mark.parametrize(
+    "vm_config_file",
+    ["framework/vm_config.json"]
+)
+def test_config_start_with_limit(test_microvm_with_api, vm_config_file):
+    """
+    Test if a microvm configured from file boots successfully.
```

Comment:
Lets add a description that's more specific to the goal of this test (i.e. testing request payload limit).